c_glib/README.md

```diff
@@ -197,3 +197,37 @@ Now, you can run unit tests by the followings:
 % cd c_glib
 % test/run-test.sh
 ```
+
+## Common build problems
+### configure failed - `AX_CXX_COMPILE_STDCXX_11(ext, mandatory)'
+
+* Check whether `autoconf-archive` is installed.
+* [macOS] Run `brew install autoconf-archive` again. If it shows like the following message, run `brew link autoconf-archive`.
+
+```text
+$ brew install autoconf-archive
+Warning: autoconf-archive 2017.03.21 is already installed, it's just not linked.
+You can use `brew link autoconf-archive` to link this version.
+```
+
+Some packages (e.g. `gnome-common`) conflict with `autoconf-archive`. If you see like the following message, run `brew unlink <pkgname>`, then run `brew link autoconf-archive` again.
```

Comment:
following -> above?

Reply:
No. `Warning: autoconf-archive 2017.03.21 is already installed, it's just not linked. ...` is a warning message of `brew install autconf-archive`. And `Linking /usr/local/Cellar/autoconf-archive/2017.03.21... ` is an error message of `brew link autoconf-archive`. So "following" is correct. But I think this paragraph may confusing, so I will rewrite.